db/db.go

```diff
@@ -0,0 +1,51 @@
+// Package db implements main options of the game through a connection to database
```

Comment:
There is an issue in the code, which can be revealed by this test

func TestDB_UserFund2(t *testing.T) {
	require := require.New(t)
	mock := New()
	require.NoError(mock.UserFund(100, 1))
}